sdk/src/types/evaluation.d.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type EvaluationSource = EASEvaluation | IPFSEvaluation;
+
+/**
+ * Schema for evaluating Hypercerts accross different sources and evaluation types
+ */
+export interface HypercertEvaluationSchema {
+  creator: string;
+  evaluationData: EvaluationData;
+  evaluationSource: EvaluationSource;
+  [k: string]: unknown;
+}
+export interface EvaluationData {
+  type: "duplicate";
```

Comment:
I assume you're imagining this to be a union type eventually? Can we also add simple text evaluations as well to help make this clear?

Reply:
Yeah good idea. Will also add an EAS schema then for signing the hash of a string (so that the text evaluation can be of arbitraty length) to keep it all 'signed with EAS'-ey

Reply:
pushed

Reply:
see [constants.ts](https://github.com/hypercerts-org/hypercerts/blob/1a45f6dc37a7e31f49305f610f035eec24e7452e/sdk/src/constants.ts), [schema](https://github.com/hypercerts-org/hypercerts/blob/1a45f6dc37a7e31f49305f610f035eec24e7452e/sdk/src/resources/schema/evaluation.json) and [implementation][https://github.com/hypercerts-org/hypercerts/blob/1a45f6dc37a7e31f49305f610f035eec24e7452e/sdk/src/evaluations/eas.ts#LL66C17-L66C17]